mission_control/navigator_alarm/navigator_alarm_handlers/kill.py

```diff
@@ -45,8 +45,8 @@ def meta_predicate(self, meta_alarm, alarms):
         ignore = []
 
         # Don't kill on low battery, only on critical
-        if alarms['battery-voltage'].raised and alarms['battery-voltage'].severity < 2:
-            ignore.append('battery-voltage')
+        #if alarms['battery-voltage'].raised and alarms['battery-voltage'].severity < 2:
```

Comment:
Not sure we should change this..

Reply:
Well right now it's broken because it uses voltage readings from killed thrusters. There is issue #254 to fix this but for now we should not use it